src/libs/Navigation/AppNavigator/getRootNavigatorScreenOptions.ts

```diff
@@ -26,7 +27,7 @@ const getRootNavigatorScreenOptions: GetRootNavigatorScreenOptions = (isSmallScr
         rightModalNavigator: {
             ...commonScreenOptions,
             cardStyleInterpolator: (props: StackCardInterpolationProps) => modalCardStyleInterpolator(isSmallScreenWidth, false, props),
-            presentation: 'transparentModal',
+            presentation: Platform.OS === 'android' ? 'modal' : 'transparentModal',
```

Comment:
this is against our code style guidelines. If you must, create a lib at `src/libs/Navigation/AppNavigator/getModalPresentationStyle` and have an android-specific implementation for just that lib.

Reply:
Done